_plugins/page_toc.rb

```diff
@@ -9,17 +9,23 @@ def initialize(tag_name, markdown, options)
 
     def render(context)
       content = context['page']['content']
-      html = ['<ul class="syn-width-full syn-overflow-hidden-x syn-scrollable-y">']
+      html = ['<nav v-scrollspy:scrollspy-content="{offset: 64}" class="syn-width-full syn-mt-4">']
+      html << '<ul class="syn-width-full syn-overflow-hidden-x syn-scrollable-y">'
       doc = Nokogiri::HTML(content)
       doc.css('h2').each do |heading|
-        html << "<li class=\"syn-overflow-hidden allow-wrap\">
-          <a class=\"flex-item-no-shrink syn-flex justify-space-between align-center syn-text-secondary\" href=\"\##{heading["id"]}\">
-            <div class=\"syn-ml-#{context['toc_indent']}\">#{heading.text}</div>
+        html << "<li class=\"sub-nav-item syn-overflow-hidden allow-wrap\">
+          <a class=\"sub-nav-link flex-item-no-shrink syn-flex justify-space-between align-center syn-text-secondary\" href=\"\##{heading["id"]}\">
+            <div class=\"sub-nav-text syn-ml-#{context['toc_indent']}\">#{heading.text}</div>
           </a>
         </li>"
       end
       html << '</ul>'
-      html.join("\n")
+      html << '</nav>'
+      if html.length > 4
```

Comment:
Alright if we add a short comment explaining the significance of 4?

Reply:
Will do!